examples/ode_models/ODE_Lotka_Volterra_multiple_ways.myst.md

```diff
@@ -0,0 +1,960 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"tags": []}
+
+(ODE_Lotka_Volterra_fit_multiple_ways)= 
+# ODE Lotka-Volterra With Bayesian Inference in Multiple Ways
+
+:::{post} January 16, 2023
+:tags: ODE, pytensor op, gradient-free inference, pytensor scan
```

Comment:
I would not use pytensor op and pytensor scan and instead use the existing [PyTensor](https://pymcio--478.org.readthedocs.build/projects/examples/en/478/blog/tag/pytensor.html) tag.

Reply:
revised